cmd/crc/cmd/daemon.go

```diff
@@ -14,6 +16,10 @@ var daemonCmd = &cobra.Command{
 	Long:   "Run the crc daemon",
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
+		// setup separate logging for daemon
+		logging.CloseLogging()
+		logging.InitLogrus(logging.LogLevel, constants.DaemonLogFilePath)
+
```

Comment:
For what it's worth, this code moving logging set up from `daemon_nonwin.go`/`daemon_windows.go` could have been done as a first commit to minimize the changes in the commit reworking the logging system.